src/sentry/runner/commands/devserver.py

```diff
@@ -72,8 +72,8 @@ def devserver(reload, watchers, workers, bind):
             raise click.ClickException('Disable CELERY_ALWAYS_EAGER in your settings file to spawn workers.')
 
         daemons += [
-            ('worker', ['sentry', 'run', 'worker', '-c', '1', '-l', 'INFO', '--autoreload']),
-            ('cron', ['sentry', 'run', 'cron', '-l', 'INFO', '--autoreload']),
+            ('worker', ['sentry', 'run', 'worker', '-c', '1', '--autoreload']),
```

Comment:
This now overrides sentry setting the log level, so if I do `devserver -l warning` celery will still log in `INFO`.